plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/markerresolution/QuickFixer.java

```diff
@@ -13,17 +15,33 @@
  * @author André Sonntag
  */
 public class QuickFixer implements IMarkerResolutionGenerator {
+	
+	ArrayList<IMarkerResolution> quickFixes;

```

Comment:
Please assign all fields a visibility keyword- whatever is the most restrictive possible.